content/en/news/2019/announcing-1.4/_index.md

```diff
@@ -11,4 +11,49 @@ aliases:
 
 We are pleased to announce the release of Istio 1.4!
 
+Version 1.4 continues our efforts at improving the user experience in Istio,
+with a focus on simplification. We've also continued to add features that
+improve the performance and experience of running Istio.
+
+## Mixer-less telemetry
+Our implementation of telemetry without Mixer will simplify installation and
+operation of meshes, all while vastly improving performance. The in-proxy
+generation of HTTP metrics has graduated from experimental to alpha. Users
+are very excited about this improvement and we are working hard to get it
+ready. We've also added new experimental features that don't require Mixer:
+TCP metrics and Stackdriver metrics.
+
+## Improved troubleshooting
+The `istioctl analyze` command continues to get new functionality for
```

Comment:
This is a new feature for 1.4.
```suggestion
We're introducing the [`istioctl analyze`](/docs/ops/diagnostic-tools/istioctl-analyze/) command to
```

Reply:
done. for some reason I thought it was in 1.3. Thanks.